migration-v4.md

```diff
@@ -248,7 +248,7 @@ module.exports = {
     client: {
       // Can be `string`:
       //
-      // webSocketURL: 'ws://0.0.0.0/ws'
+      // webSocketURL: 'ws://0.0.0.0:8080/ws'
```

Comment:
We can use here `:0` to get port from browser, I think better to indicate this

Reply:
Like this ?

```suggestion
      // use ":0" to get port from the browser
      // webSocketURL: 'ws://0.0.0.0:0/ws'
```

Reply:
It is already written somewhere here. maybe maybe will be duplicate it, `auto://0.0.0.0:0/ws` get protocol/hostname/port from browser